### PR TITLE
[7.0.x] PLANNER-666: Execution server: Simplify response XML for OptaPlanner REST calls

### PR DESCRIPTION
**Pull request by mcimbora**

### CODE REVIEW

**Comment by mcimbora** [comment]
Jenkins retest this please

**Comment by mcimbora** [comment]
Jenkins retest this.

File: kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java [hunk 1]
```diff
@@ -38,11 +40,6 @@
     @XStreamAlias("score")
     private ScoreWrapper scoreWrapper;
 
-    @XmlElement(name = "planning-problem")
-    @XStreamAlias("planning-problem")
-    @XmlJavaTypeAdapter(JaxbUnknownAdapter.class)
-    private Object planningProblem;
-
```
Comment (solutionjh) [inline kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java]: why delete planning-problem?
I use [POST] to execute optaplanner, but there occurs 500 Internal Server Error
they say there is no instance SolverInstance.planning-problem.

File: kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java [hunk 2]
```diff
@@ -87,18 +84,10 @@ public ScoreWrapper getScoreWrapper() {
...
     }
 
-    public Object getPlanningProblem() {
-        return planningProblem;
-    }
-
-    public void setPlanningProblem(Object planningProblem) {
-        this.planningProblem = planningProblem;
-    }
-
```
Comment (solutionjh) [inline kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java]: why delete planning-problem?
I use [POST] to execute optaplanner, but there occurs 500 Internal Server Error
they say there is no instance SolverInstance.planning-problem.

File: kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java (hunk 1 above)
Comment (mcimbora) [inline kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java]: @solutionjh 
The Kie Server API has changed in 7.0.x, please see https://github.com/kiegroup/kie-docs/blob/master/docs/optaplanner-wb-es-docs/src/main/asciidoc/Workbench/Quickstart/Quickstart-section.adoc#submit-solution to check how to submit a planning problem in a new way.

Register a solver:
PUT http://${kie-server}/services/rest/server/containers/${container_id}/solvers/${solver_id}

Submit a solution:
POST http://${kie-server}/services/rest/server/containers/${container_id}/solvers/${solver_id}/state/solving

Get the best solution:
GET http://${kie-server}/services/rest/server/containers/${container_id}/solvers/${solver_id}/bestsolution

Terminate the solver:
POST http://${kie-server}/services/rest/server/containers/${container_id}/solvers/${solver_id}/state/terminating-early

Dispose a solver:
DELETE http://${kie-server}/services/rest/server/containers/${container_id}/solvers/${solver_id}

Regards,
Matej

Comment (solutionjh) [inline kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java]: @mcimbora 
Thank you !!^^
I read only manual in drools web page, so I can't find this change.
New method work well^^

**Pull request merged**